.devcontainer/scripts/onCreateCommand.sh

```diff
@@ -35,17 +35,6 @@ REQUIREMENTS="./app/requirements-links.txt"
 if [ -f $REQUIREMENTS ]; then
     pip3 install -r $REQUIREMENTS
 fi
-# Dependencies for the app
-REQUIREMENTS="./app/requirements.txt"
-if [ -f $REQUIREMENTS ]; then
-    pip3 install -r $REQUIREMENTS
-fi
-
-# Dependencies for unit and integration tests
-REQUIREMENTS="./app/tests/requirements.txt"
-if [ -f $REQUIREMENTS ]; then
-    pip3 install -r $REQUIREMENTS
-fi
```

Comment:
did you check that all requirements are included in the main requirements.txt? A quick check already revealed that e.g. dapr is named in the app/requirements.txt but not in the root requirements.txt...